### PR TITLE
FHIR-43324 Added measurereport-populationDescription extension

### PR DESCRIPTION
**Pull request by dand9959**
Controlling FHIR ticket: https://jira.hl7.org/browse/FHIR-43324

Added a measurereport-populationDescription to the registry.  

(The aforementioned FHIR ticket's comments indicate that the extension might be added directly to the DEQM IG on an interim basis to avoid display errors while referencing the extension in the measureReport profiles.  After further discussion, it was decided to, instead, add the extension to this registry IG as described in the ticket's resolution, accepting the fact that those display errors will be present in the DEQM IG until this registry is next published. )

### CODE REVIEW

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    "date" : "2024-02-01",
+    "publisher" : "HL7 International / Clinical Quality Information WG",
+    "contact" : [{
+      "telecom" : [{
+        "system" : "url",
+        "value" : "http://www.hl7.org/Special/committees/cqi"
+      }]
+    }],
+    "description" : "The human readable description of population criteria.",
+    "jurisdiction" : [{
```
Comment (brynrhodes) [inline input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json:30]: I wouldn't limit the jurisdiction to the US, this is just a general extension for use in describing population criteria in a measure report

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      "expression" : "MeasureReport.supplementalData"
+    }],
+    "type" : "Extension",
+    "baseDefinition" : "http://hl7.org/fhir/StructureDefinition/Extension",
+    "derivation" : "constraint",
+    "differential" : {
+      "element" : [{
+        "id" : "Extension",
+        "path" : "Extension",
+        "short" : "Provides a description of the population in which it appears.",
```
Comment (brynrhodes) [inline input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json:68]: "on which it appears"

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        "definition" : "Provides a description of the population in which it appears.",
```
Comment (brynrhodes) [inline input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json:69]: More detail here, should probably be:
"A description of the group, population, stratifier, or supplemental data definition on which it appears. This is typically a restatement of the description of the corresponding element in the Measure being reported, but may also include additional information gathered by the reporting process."

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        "min" : 0,
+        "max" : "1"
+      },
+      {
+        "id" : "Extension.url",
+        "path" : "Extension.url",
+        "type" : [{
+          "code" : "uri"
+        }],
+        "fixedUri" : "http://hl7.org/fhir/extensions/StructureDefinition-measurereport-populationDescription"
```
Comment (brynrhodes) [inline input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json:79]: This fixedUri is incorrect, it needs to match the URL of the extension structure definition:
"http://hl7.org/fhir/StructureDefinition/measurereport-populationDescription"

**Review by brynrhodes — CHANGES_REQUESTED** [review]

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json (hunk 1 above)
Comment (dand9959) [reply input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json:30]: Latest commit: 6065105
Used the  http://unstats.un.org/unsd/methods/m49/m49.htm jurisdiction, 001, "World" as used in other extensions

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json (hunk 2 above)
Comment (dand9959) [reply input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json]: Latest commit: 6065105

Changed phrasing as indicated throughout.

File: input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json (hunk 4 above)
Comment (dand9959) [reply input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json]: Latest commit: 6065105
Corrected URI as described.

**Review by grahamegrieve — APPROVED** [review]
ok

**Pull request merged**